alternatingCase/alternatingCase.test.js

```diff
@@ -0,0 +1,6 @@
+var alternatingCase = require('./alternatingCase');
```

Comment:
Create a separate folder for test files and put all your test files inside that folder, not in the same folders as your code implementations.

Reply:
What if I just create on universal test file and then import all my methods there and then create all the tests in that file